cloudbuild.yaml

```diff
@@ -68,6 +68,13 @@ steps:
 
         gcloud compute url-maps invalidate-cdn-cache $_SERVICE_NAME-$$CLUSTER_NAME-https-lb --async --path "/index.html"
         gcloud compute url-maps invalidate-cdn-cache $_SERVICE_NAME-$$CLUSTER_NAME-https-lb --async --path "/*"
+      elif [ "$BRANCH_NAME" = "nikolay" ]; then
+        gsutil -m -h "Cache-Control:public,max-age=1209600" cp -z html,js,css,svg -R * gs://$$PROJECT_NAME-$$CLUSTER_NAME-$_SERVICE_NAME-$BRANCH_NAME
+        gsutil setmeta -h "Cache-Control:private, max-age=0, no-transform" gs://$$PROJECT_NAME-$$CLUSTER_NAME-$_SERVICE_NAME-$BRANCH_NAME/index.html
```

Comment:
@Goginet do we have a command to create google storage in devops repo?
https://github.com/UnFederalReserve/devops/pull/30

Reply:
I think we don't need commit this changes to devops because this is temporary brunch? 
But if you think it's necessary - https://github.com/UnFederalReserve/devops/pull/31